src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -320,5 +320,79 @@ limitations under the License.
     </ACTION>
     </COMMAND>
 
+<!--====================clear mac address-table dynamic <all | vlan | address>======================-->
+
+    <COMMAND
+        name="clear mac"
+        help="clear mac"
+        >
+    <ACTION>
```

Comment:
Doesnt this allow to execute "clear mac" command with no other arguments?

Reply:
Following OS10 CLI.

OS10# clear mac address-table dynamic 
% Error: The command is not completed.
OS10# 

So with "clear mac address-table dynamic all" will clear dynamic FDB entries.